internal/controller/apiextensions/definition/reconciler.go

```diff
@@ -115,10 +124,18 @@ func (fn CRDRenderFn) Render(d *v1.CompositeResourceDefinition) (*extv1.CustomRe
 func Setup(mgr ctrl.Manager, o apiextensionscontroller.Options) error {
 	name := "defined/" + strings.ToLower(v1.CompositeResourceDefinitionGroupKind)
 
-	r := NewReconciler(mgr,
+	r := NewReconciler(mgr, o,
 		WithLogger(o.Logger.WithValues("controller", name)),
-		WithRecorder(event.NewAPIRecorder(mgr.GetEventRecorderFor(name))),
-		WithOptions(o))
+		WithRecorder(event.NewAPIRecorder(mgr.GetEventRecorderFor(name))))
+
+	if o.Features.Enabled(features.EnableRealtimeCompositions) {
+		if err := mgr.Add(manager.RunnableFunc(func(ctx context.Context) error {
+			wait.UntilWithContext(ctx, r.xrInformers.reconcileManagedResourcesInformers, time.Minute)
```

Comment:
Maybe add a comment explaining that `reconcileManagedResourcesInformers` will run every minute until the context is cancelled. As someone who doesn't use `util/wait` very often I suspected that was the case but had to go look at its documentation to confirm. It's not super obvious from the function name/signature - "wait" doesn't sound like "run this repeatedly".

Reply:
done